apluslms_shepherd/celery_tasks/build/tasks.py

```diff
@@ -145,5 +137,5 @@ def clean(res, base_path, course_key, instance_key, build_number):
 def error_handler(uuid):
     result = AsyncResult(uuid)
     exc = result.get(propagate=False)
-    print('Task {0} raised exception: {1!r}\n{2!r}'.format(
+    logger.warning('Task {0} raised exception: {1!r}\n{2!r}'.format(
```

Comment:
You should use  `%s` and logger method arguments with logging module. That way, the formating is not done, unless the line is actually logged somewhere.

Reply:
I'll chang every log to the format like 
```
logger.info("Current build numbers: %s", tuple(number_list))
``` 
For performance reason.
ref. : https://stackoverflow.com/questions/11955787/idiomatic-python-logging-format-string-args-list-vs-inline-string-formatting

Reply:
conversion from list to tuple is also some extra work, but otherwise yep, that is the way :)